Documentation/input/devices/edt-ft5x06.rst

```diff
@@ -51,4 +51,4 @@ raw_data:
 
 Note that reading raw_data gives a I/O error when the device is not in factory
 mode. The same happens when reading/writing to the parameter files when the
-device is not in regular operation mode.
+device is not in regular operation mode.
```

Comment:
Seems like nothing was changed here. Should we remove this 'change'?

Reply:
My editor changed the line ending, it does no harm and it is not very trivial to correct it.